pwndbg/dbg/gdb.py

```diff
@@ -17,6 +17,64 @@
 from pwndbg.gdblib import load_gdblib
 
 
+def parse_and_eval(expression: str, global_context: bool) -> gdb.Value:
+    """
+    Same as `gdb.parse_and_eval`, but only uses `global_context` if it is
+    supported by the current version of GDB.
```

Comment:
Pls add info what GDB version allows for `global_context`

Reply:
I've added it, now.